python/nav/ipdevpoll/shadows/__init__.py

```diff
@@ -287,9 +287,8 @@ def prepare(self, containers):
                 self.net_type = net_type
 
     def save(self, containers):
-        if (self._has_no_prefixes(containers) or
-                self._revert_vlan_on_type_change_to_scope(containers) or
-                self._is_type_changed_to_static(containers)):
+        if self._revert_vlan_on_type_change_to_scope(containers) or \
```

Comment:
I don't have any such preference in general. But for if statements, if you use parenthese the next line of the if will get indented four spaces and line up perfectly with the body of the if, and this is bad for readability, and which flake8 complains about. As for this particular case the continuation lines were overindented possibly to avoid that and it just looks weird to me.